Library/Homebrew/cask/lib/hbc/cli/outdated.rb

```diff
@@ -0,0 +1,39 @@
+module Hbc
+  class CLI
+    class Outdated < Base
+      def self.run(*args)
+        greedy  = args.include?("--greedy")
+        verbose = ($stdout.tty? || ARGV.verbose?) && !args.include?("--quiet")
+
+        cask_tokens = cask_tokens_from(args)
+        casks_to_check = if cask_tokens.empty?
+          Hbc.installed
+        else
+          cask_tokens.map { |token| Hbc.load(token) }
+        end
+
+        casks_to_check.each do |cask|
+          odebug "Checking update info of Cask #{cask}"
+          list_if_outdated(cask, greedy, verbose)
+        end
+      end
+
+      def self.list_if_outdated(cask, greedy, verbose)
+        return unless cask.outdated?(greedy)
+
+        if verbose
+          outdated_versions = cask.outdated_versions(greedy)
+          outdated_info   = "#{cask.token} (#{outdated_versions.join(", ")})"
+          current_version = cask.version.to_s
+          puts "#{outdated_info} < #{current_version}"
```

Comment:
I understood the reasons about using `!=` to compare the versions and I changed the code. 

Please remark the `puts` in front of that line. It is not a version comparison, `"<"` is part of the text that is displayed to the user:

    $ brew cask outdated
    cyberduck (5.3.7.23440, 5.3.8.23611) < 5.3.9.23624
    virtualbox (5.1.14-112924) < 5.1.16-113841
    virtualbox-extension-pack (5.1.14-112924) < 5.1.16-113841